actions/pack_action.go

```diff
@@ -70,9 +72,18 @@ func (pf *PackAction) Run(context *debos.DebosContext) error {
 	pf.LogStart()
 	outfile := path.Join(context.Artifactdir, pf.File)
 
-	var tarOpt = "cf" + tarOpts[pf.Compression]
 	log.Printf("Compressing to %s\n", outfile)
-	return debos.Command{}.Run("Packing", "tar", tarOpt, outfile,
-		"--xattrs", "--xattrs-include=*.*",
-		"-C", context.Rootdir, ".")
+	var tarOpt = "cf"
```

Comment:
looks like it would be cleaner if you took a similar approach to how you call tar it in the unpack_action